server/controllers/work.js

```diff
@@ -1,17 +1,69 @@
 import {createPageConfig} from '../model/page-config';
-import {getWork} from '../services/wellcomecollection-api';
+import {getWork, getWorks} from '../services/wellcomecollection-api';
+import {createResultsList} from '../model/results-list';
+import {PaginationFactory} from '../model/pagination';
+import {List} from 'immutable';
+
+function imageUrlFromMiroId(id) {
+  const cleanedMiroId = id.match(/(^\w{1}[0-9]*)+/g, '')[0];
+  const miroFolder = `${cleanedMiroId.slice(0, -3)}000`;
+  return `http://s3-eu-west-1.amazonaws.com/miro-images-public/${miroFolder}/${id}.jpg`;
```

Comment:
I know this is just for a prototype, but seeing this URL here makes me feel like we should have a domain name redirecting to the image bucket, e.g. `i.wellcomecollection.org`